Threats

stfinnis opened the pull request:
Adds Threats section to the Report.
A few things to look at:

- A few tables are maybe too wide? I didn't adjust any of the settings. Take a look and see what you think
- The Notes below two of the Threat Assessment tables are in grey. Should they be black to match the Table caption text above the table?
- I don't know how you feel about blank spaces in the tables. It's kind of weird. For example, this is all of the information for Acadian Redfish. But then the Atlantic COSEWIC population has three RPAs. Is there a better way to do this?

stoyelq commented:
I played around a bit with the tables, it all happened in the excel sheet though.  At the moment, I only made changes to the NonSchedule1Documents sheet, namely removing underscores from the column headers, added COSEWIC Status Report to the SAR Public Registry header, and consolidating all the RPA's into single cells.  This makes all the rows a little larger, but dumps a lot of the empty space: 
![image](https://user-images.githubusercontent.com/74926142/161259012-f456271d-f77f-4b1c-9bf8-469adfc663a3.png)

Concerning the table width, table 2 is a little wide but the only thought I have there is to stick all the RPA's and COSEWIC Status Reports in one table (regardless of status) and all the schedule 1 documents in the other.  

All the code looks great, the last thing to do is to add a call to Rmd so it gets integrated into the main report, probably into app/templates/EN/human_threats/miscellaneous.Rmd, take a look at app/templates/EN/human_threats/fishing.Rmd or any of the other templates to see what you need.

stfinnis commented:
Thanks! The new formatting looks great. 
I think to reduce the width of the Schedule 1 table, the Threat Assessment column can be removed. These are provided as separate Tables. I added a note about this in the Table caption.
I have also added a call to app/templates/EN/human_threats/miscellaneous.Rmd to integrate this section into the main report
